matlab/setupProcParams.m

```diff
@@ -7,37 +7,16 @@ function setupProcParams()
 % output:
 % Output is assigned to workspace variables.
 %
-% Note, SPM.mat structure is prepared running specified SPM batch module
-% (fMRI model specification), on functional localizer data
-% of the same/pilot subject. Some parameters could be set constant for
-% the whole study, for simplicity, keeping in mind their potential 
-% variability, e.g. the TH values.
-% Note that this is not a perfect solution and requires analytical and 
-% programming efforts in future studies. Perspectively, the TH value 
-% calculation algorithm (as in SPM) has to be implemented.
-% An encouraged user could explore SPM.xM.TH estimations and advance
-% iGLM estimation scheme further.
-%
-% For now, note that spmMaskTh is assigned from user-specified SPM  
-% structure and SPM.xM.TH values, for simplicity, neglecting the temporal  
-% variability (std ca. 0.5%), see in code. 
-% This is because SPM provides the TH value for each volume given the data,
-% which is not available in real-time in the current OpenNFT version. 
-% TH values could depend on the data acquisition setup, and could evtl. be
-% set as mean(SPM.xM.TH)*ones(size(SPM.xM.TH)), or zeros(size(SPM.xM.TH)) 
-% given masking threshold defined in SPM batch. 
 % The iGLM estimations are used for visualizations, however, note that
 % negligible variations are possible in dynamic ROI update schemes or
 % feedback estimations based on iGLM. 
 %
 % Note, the iGLM/cGLM contrasts are hard-coded and user-/study- defined,
 % which is linked to the prepared SPM.mat structure.
-% Note that for simplicity, intermittent time-series settings are used for 
-% continuous feedback demonstration.
 % An end-user needs to set and justify their own parameter
 % files and contrasts.
 %__________________________________________________________________________
-% Copyright (C) 2016-2019 OpenNFT.org
+% Copyright (C) 2016-2017 OpenNFT.org
```

Comment:
Please fix copyright to "2016-2019" here and further.

Reply:
Thanks!